/**:
  ros__parameters:
    max_cluster_size: 1000
    min_cluster_size: 10
    target_frame: base_link
    tolerance: 0.7
    use_height: false
